csv-bulk-signature/src/main.ts

```diff
@@ -222,6 +220,16 @@ function skipFirstPlaceholderRow(contact: Contact) {
   return contact["Brand*"] !== "use dropdown to select a brand";
 }
 
+export function filterFilesForZip(file: string) {
+  return file.endsWith(".htm") || file.endsWith(".txt");
+}
+
+export const checkRequiredFields = (row: Contact) =>
+  !!row["Brand*"]?.length &&
+  !!row["Full Name*"]?.length &&
+  !!row["Title*"]?.length &&
+  !!row["Office Phone*"]?.length;
```

Comment:
add optional chaining for if one required field doesn't exist and we can't get the length of it